api-reference/integrations/update-integration.mdx

```diff
@@ -1,3 +1,75 @@
 ---
 openapi: put /v1/integrations/{integrationId}
----
\ No newline at end of file
+---
+
+<RequestExample>
+
+```javascript Node.js
+import { Novu } from '@novu/node';  
+export const novu = new Novu('<REPLACE_WITH_API_KEY_FROM_ADMIN_PANEL>');
+await novu.integrations.getWebhookProviderStatus(ProvidersIdEnum.SendGrid);
+```
+
+```php PHP
+// Get integrations
+use Novu\SDK\Novu;
+
+$novu = new Novu(<NOVU_API_KEY>);
+$novu->updateIntegration($integrationId, [
+    'active' => true,
+    'credentials' => [
+        // insert all the fields
+    ],
+    'check' => true
+])->toArray();
+```
+
+```ruby Ruby
+require 'novu'
+
+client = Novu::Client.new('<NOVU_API_KEY>')
+body = {
+    'active' => true,
+    'check' => true
+    'credentials' => {
+        # insert all the fields
+    },
+}
+client.update_integration('<insert-provider-id>', body)
+```
+
+```python Python
+from novu.api import IntegrationApi
+
+url = "https://api.novu.co"
+api_key = "<NOVU_API_KEY>"
+novu = IntegrationApi(url, api_key).update(
+    integration="<The instance of the integration to update>",
+)
+```
+
+```java Java
+import co.novu.sdk.Novu;
+
+public class Main {
+    public static void main(String[] args) {
+        String apiKey = "<NOVU_API_KEY>";
+
+        Novu novu = new Novu(apiKey);
+        Novu.updateIntegration(integrationId, body);
```

Comment:
`Novu` should be `novu` because you are meant to use the object/instance of the class.